mockOSApi.v2/Models/OSObject.cs

```diff
@@ -2,54 +2,62 @@
 
 namespace mockOSApi.Models;
 
-public abstract class OSObject {
+public abstract class OSObject
+{
 
 
     /* Fields and properties
     *******************************
     */
     private int _handle;
```

Comment:
This is all equivalent to `public int Handle { get; set; }`
Unless you have some custom code involved, don't declare the private int property separately.

Reply:
removed this